Library/Homebrew/cmd/pull.rb

```diff
@@ -189,8 +189,14 @@ def pull
               "-u#{bintray_user}:#{bintray_key}", "-X", "POST",
               "https://api.bintray.com/content/homebrew/#{repo}/#{package}/#{version}/publish"
             puts
-            sleep 20
-            safe_system "brew", "fetch", "--retry", "--force-bottle", f.full_name
+          end
+          unless (Pathname.pwd/".git/hooks/pre-push").exist?
+            opoo "Pre-push hook missing"
+            puts <<-EOS.undent
+              Please consider symlinking #{HOMEBREW_CONTRIB}/pre-push into
+              #{Pathname.pwd}/.git/hooks to add a check that bottles are published
+              correctly before pushing to master.
+            EOS
```

Comment:
To make less surprising, we could add `brew pull --install-push-hook` or something?